net/samba4/Config.in

```diff
@@ -0,0 +1,111 @@
+if PACKAGE_samba4
+
+config SAMBA4_SERVER
+	bool "Server (fileserver)"
+	select BUSYBOX_CONFIG_RENICE
```

Comment:
This is a bad idea, don't modify other package "indirectly". Submit that it should be enabled by default upstream instead if it's really needed however my guess is that it's not the end of the world if renice isn't available.

Reply:
Probably true, i think it was originally used to prevent the samba process itself to block other network related work, so it does not stall itself. I never tested it without renice, that's why i kept it in and found no other way to include it.
So there is no "official" way to have a package depend on a specific config option? How do other packages handle this, say for example if they depend on a kernel mod or specific kernel option?

Reply:
You can depend on kernel mods however there's no good way of depending on options that doesn't result in a unique package to my knowledge. @karlp @hnyman